.github/workflows/a11y-test.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  run_tests:
+    name: a11y
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [ 3.8 ]
+        node-version: [ 12 ]
+        os: [ ubuntu-latest ]
```

Comment:
Can we make this `20.04` explicitly. So we can have more control over the upgrade.

Reply:
updated the matrix as per suggestion.